evap/results/templates/results_evaluation_detail.html

```diff
@@ -32,68 +32,74 @@
-                                        {% translate 'Shows results available for everyone logged in.' %}"
-                                        >
-                                        {% translate 'Public' %}
-                                    {% endif %}
-                            </a>
-                        {% endif %}
+                    <div class="row">
+                        <div class="col-auto">
+                            <div class="btn-switch btn-switch-light my-auto d-print-none">
+                                <div class="btn-switch-label">{% translate 'General results' %}</div>
+                                <div class="btn-switch btn-group">
+                                    <a
+                                        href="{% url 'results:evaluation_detail' evaluation.course.semester.id evaluation.id %}?view_general_results=full&view_contributor_results={{ view_contributor_results.value }}"
+                                        role="button"
+                                        class="btn btn-sm btn-light {% if not general_textanswers %} disabled {% elif view_general_results == ViewGeneralResults.FULL %} active{% endif %}"
+                                        data-bs-toggle="tooltip"
+                                        data-bs-placement="bottom"
+                                        title="{% blocktranslate %}All results of general questions (including text answers){% endblocktranslate %}"
+                                    >
+                                        {% translate 'Full' %}
+                                    </a>
+                                    <a
+                                        href="{% url 'results:evaluation_detail' evaluation.course.semester.id evaluation.id %}?view_general_results=ratings&view_contributor_results={{ view_contributor_results.value }}"
+                                        role="button"
+                                        class="btn btn-sm btn-light {% if not general_textanswers %} disabled {% elif view_general_results == ViewGeneralResults.RATINGS %} active {% endif %}"
+                                        data-bs-toggle="tooltip"
+                                        data-bs-placement="bottom"
+                                        title="{% translate 'Only rating results of general questions (without text answers)' %}"
+                                    >
+                                        {% translate 'Ratings' %}
+                                    </a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                
+                <div class="row mt-1">
+                    <div class="col-auto">
+                        <div class="btn-switch btn-switch-light my-auto d-print-none">
+                            <div class="btn-switch-label">{% translate 'Contributor results' %}</div>
```

Comment:
when disabled, "Ratings" should be selected/active and its tooltip should be shown on hover.
a different tooltip should be shown for the other buttons when disabled: "You can't see text answers for contributor questions because you aren't a contributor."

Reply:
We added that ratings is selected when the button group is disabled.
Showing the tooltip is not this easily possible since the buttons are disabled and therefore dont accept any mouse events (inlcuding mouse hover), thus the title from the django template is not shown.

Reply:
I think we found a way of having tooltips on disabled buttons in https://github.com/e-valuation/EvaP/pull/1931.